Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,182 @@
+description: Integration to fetch and update symantec DLP Incidents
+detaileddescription: Symantec DLP integration uses SOAP API to communicate with Enforce
+  server to fetch and update  incident Data. uses Zeep package for SOAP Api
+configuration:
+- display: 'credentials '
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: 'Server URL'
+  name: server-ip
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: 'Saved Report ID'
+  name: saved-Report-id
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: |+
+    from requests import Session
+    from zeep import Client
+    from zeep.transports import Transport
+    from requests.auth import AuthBase, HTTPBasicAuth
+    from zeep import helpers
+    import random
+    import datetime
+    import json
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+    requests.packages.urllib3.disable_warnings()
+
+    # hexdump:732073756468656572206b756d6172
+
+    class SymantecAuth(AuthBase):
+        def __init__(self, USERNAME, PASSWORD, host):
+            self.basic = HTTPBasicAuth(USERNAME, PASSWORD)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    SERVER_IP = demisto.params()['server-ip']
+    USERNAME = demisto.params()['credentials']['identifier']
+    PASSWORD = demisto.params()['credentials']['password']
+    SAVED_REPORT_ID = demisto.params()['saved-Report-id']
+
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(SERVER_IP)
+
+    session = Session()
+    session.auth = SymantecAuth(USERNAME,PASSWORD, "https://{}".format(SERVER_IP))
+    session.verify = False
+    transport = Transport(session=session)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  get_incident_id(dtobj):
+
+        if type(dtobj) is datetime.datetime:
+            return client.service.incidentList(savedReportId=SAVED_REPORT_ID, incidentCreationDateLaterThan=dtobj)
+        else:
+            return_error('excpected datetime object')
+
+
+
+    def incidentDetail(_id):
+
+        return client.service.incidentDetail(incidentId=_id,includeViolations=True,includeHistory=True)
+
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            last_run['lt_time'] = datetime.datetime.strptime("01/11/2018 00:00:00.000000",'%d/%m/%Y %H:%M:%S.%f')
```

Comment:
"01/11/2018 00:00:00.000000" is hardcoded date.
For the first fetch we usually add integration parameter `Fetch since` and the value could `2 days`, `1 month`, etc. You can use as an example https://github.com/demisto/content/blob/master/Integrations/integration-OTRS.yml#L59

Reply:
@anara123  one doubt does "parse_date_range( )" already implemented in Demisto or do I need to explicitly implement it cause in the example integration you mentioned, I could not find it implemented anywhere
if it is already implemented can I get a reference for that

Reply:
@ssudheerk99 It is already implemented in our `CommonServerPython` script, which you can find here https://github.com/demisto/content/blob/master/Scripts/script-CommonServerPython.yml

Reply:
@Itay4 thanks for the reference I have implemented the method before I saw you I comment I let that it be as both seems to be same :-p .
@anara123  I have added the fetch time parameter

let me know if any more changes to be made :-)